### PR TITLE
Remove Yarn

### PR DESCRIPTION
**Pull request by alexwllms**
Remove Yarn from the project:

- Update Travis file to use NPM
- Update Readme file
- Remove yarn.lock

### CODE REVIEW

File: src/CheckBox/__tests__/__snapshots__/CheckBox.js.snap [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 exports[`renders 1`] = `
 <label
-  class="sc-htpNat gAjghl"
+  class="sc-htpNat fxMOOa"
```
Comment (txwkx) [inline src/CheckBox/__tests__/__snapshots__/CheckBox.js.snap]: 🤔 hmm

File: README.md [hunk 2]
```diff
@@ -24,9 +24,7 @@ components that conform to Marshmallow's Design System - "_S'mores_".
 ## Install
 
 ```bash
-$ yarn add @mrshmllw/smores-react
```
Comment (txwkx) [inline README.md]: moving the project back to `npm` wouldn't prevent people from installing it using `yarn`

Comment (txwkx) [reply README.md]: shall we leave the **README.md** as it is?

File: src/CheckBox/__tests__/__snapshots__/CheckBox.js.snap (hunk 1 above)
Comment (txwkx) [reply src/CheckBox/__tests__/__snapshots__/CheckBox.js.snap]: need to tackle those fake snapshot changes asap

File: README.md (hunk 2 above)
Comment (alexwllms) [reply README.md]: That makes sense, although we should punish them.

**Pull request merged**